package.json

```diff
@@ -199,6 +201,7 @@
     "dev:client": "NODE_ENV=development webpack --progress --profile --watch --config webpack/client/webpack.config.js",
     "format": "eslint --fix '**/*.{ts,tsx,js,jsx}'",
     "lint": "eslint '**/*.{ts,tsx,js,jsx}'",
+    "storybook": "start-storybook --ci -p 6001",
```

Comment:
I'd like to run `storybook` script when run `dev` script. What do you think?

Reply:
I overlooked `dev` script… It's a great idea!!!